settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:console_authorize_with_ssl_client: true
+:console_authorize_with_trusted_hosts: false
```

Comment:
Yes. First we thought that we don't want to make the `trusted_hosts` check turned on by default, as the admin could access the host from differen host, than the foreman one, on the other hand, the proxy uses just the cn in the cert (and a user could even add their CN into the trusted hosts if needed), which is better, than trusting anyone with a valid cert (even though just puppet client). So +1 for just on/off for authorizing the console access and using both trusted host and certs mechanism